### PR TITLE
Adding runner + rundeck self signed in how to section.

### PR DESCRIPTION
**Pull request by jplassnibatt**
Adding runner-paop-selfsigned.md.
Adding an entry in the admin list.
Editing an entry in the admin list.
I didn't know what other files to touch to add the entry correctly.

### CODE REVIEW

File: docs/learning/howto/index.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Available Guides by Section
 [How to Manage Jobs as Code](/learning/howto/how2scm.md)  
 [Configure Rundeck behind an egress proxy](/learning/howto/egress-proxy.md)  
 [Deploy Rundeck on AWS with Terraform](/learning/howto/how2-terra-rd-aws.md)  
-[Deploy Rundeck on AWS EKS with Terraform](/learning/howto/how2-terra-rd-eks.md)  
+[Deploy Rundeck on AWS EKS](/learning/howto/how2-terra-rd-eks.md)
```
Comment (fdevans) [inline docs/learning/howto/index.md:29]: Was this change intentional as part of this PR?

**Review by MegaDrive68k — APPROVED** [review]
LGTM.

File: docs/learning/howto/index.md (hunk 1 above)
Comment (jplassnibatt) [reply docs/learning/howto/index.md:29]: Yes, part of the "Editing an entry in the admin list.", that one is not for Terraform.
